FBSnapshotTestCase/UIImage+Compare.m

```diff
@@ -76,15 +84,53 @@ - (BOOL)compareWithImage:(UIImage *)image
     free(imagePixels);
     return NO;
   }
-    
+
+    /*
+  CGFloat scaleFactor = [[UIScreen mainScreen] scale];
+  CGContextScaleCTM(referenceImageContext, scaleFactor, scaleFactor);
+  CGContextScaleCTM(imageContext, scaleFactor, scaleFactor);
+    */
```

Comment:
My guess is that scaling changes the number of pixels that the algorithm below needs to enumerate.